src/snowflake/cli/api/project/definition_conversion.py

```diff
@@ -130,7 +131,10 @@ def convert_project_definition_to_v2(
         # since the file won't be re-read as it would be for a permanent conversion
         definition_v2 = render_definition_template(data, {}).project_definition
     else:
-        definition_v2 = ProjectDefinitionV2(**data)
+        # Exact values of context are irrelevant.
+        # It just can't be none, so the mixins are applied
```

Comment:
isn't it odd to have this fake context with a dummy key:value pair? Could we at least make it work with an empty dict or something?

Reply:
Changed to `{}` - this also cause changed in the condition in `apply_mixins`